README.md

```diff
@@ -9,6 +9,21 @@ Setup
 
 3. Run `gatsby develop`. This will start a hot-reloading development environment at `localhost:8000`.
 
+Run 2018 site locally
+-----------------------------
+1. Run `npm run build2018`. This will:
+   1. Remove any current `2018` folder in the `static` directory
+   2. Go into /2018
+   3. Install dependencies
+   4. Build the 2018 survey site
+   5. Copy that build into `static` directory
+
+2. Run `gatsby build` to build the 2019 site.
+
+3. Run a local server. Options for this include:
+   1. Python server, run `python -m SimpleHTTPServer 8000`
+   2. npm server, run `npm i -g http-server`
```

Comment:
I'm having issues running the 2018 version of the site. It could be user error. 😆

It doesn't look like the CSS builds in step 1 above. This is what I see when I run `python -m SimpleHTTPServer 8000`:

<img width="839" alt="2018" src="https://user-images.githubusercontent.com/12678977/58261169-cef6b180-7d45-11e9-9669-59f75da8d281.png">


<img width="839" alt="2018" src="https://user-images.githubusercontent.com/12678977/58261065-a078d680-7d45-11e9-80ff-52fc3909e12d.png">